### ISSUE TITLE
wrong ID for MYC (gene) test

### ISSUE THREAD

**Issue by colleenXu** [NCATSTranslator/Tests#39]
The "NeverShow: cancer decreases activity or abundance of MYC proto-oncogene, bHLH transcription factor" test (Test case 24, asset 312) is using the gene ID NCBIGene:7553. I can see this in the test log https://informationradiator.renci.org/test-runs/53/tests/9184

However, [this ID is actually for ZNF7](https://www.ncbi.nlm.nih.gov/gene/7553). Instead, I think [NCBIGene:4609](https://www.ncbi.nlm.nih.gov/gene/4609) is the ID for "MYC proto-oncogene, bHLH transcription factor".

**Comment by maximusunc** [comment]
I definitely agree that the curie does not match the name. @sandrine-m do you think you could look into this and potentially use NCBIGene:4609 for this asset? I ran it through node norm and it just said MYC, so not sure if that's close enough.

**Comment by sandrine-muller-research** [comment]
Thanks @colleenXu  for reporting this asset. @maximusunc I have not submitted this particular test asset but I can try to have a look at it to try to understand what the user meant (MYC or something else?)

**Comment by sandrine-muller-research** [comment]
corrected with the proposed NCBIGene:1636 ID in the test asset sheet. Please close if the error does not happen anymore. Thanks

**Comment by colleenXu** [comment]
[NCBIGene:1636](https://www.ncbi.nlm.nih.gov/gene/1636) is ACE. So is this meant to be one of the ACE tests (Test case 23 in the [sheet](https://docs.google.com/spreadsheets/d/1x1BvzuU5EDmovvkaVAT6yG7-HC7lkgK8hELMqwoHNNQ/edit#gid=1469161327)?

**Comment by sandrine-muller-research** [comment]
Thank you @colleenXu for correcting me. I do not know what are ACE tests.
I made a mistake when I responded to you earlier and did not copy paste the good ID in my answer on GitHub. I verified today that the correction I made earlier is the correct ID : NCBIGene:4609 for MYC. I corrected in the [Testing QA Pairs](https://docs.google.com/spreadsheets/d/1wAQaFEtFqAvp2fbTZIe-2ObF9zUU_cmXILfU8SzUWe0/edit?usp=sharing) from which the test cases are built. I did not edit another sheet. I am not sure if I answered your question?
